frontend/src/app/containers/ExperimentPage/stories.js

```diff
@@ -69,3 +156,104 @@ storiesOf('ExperimentPage/TestpilotPromo', module)
   .add('has add-on', () =>
     <TestpilotPromo {...{ ...testpilotPromoBaseProps, hasAddon: true }} />
   );
+
+storiesOf('ExperimentPage/EolBlock', module)
```

Comment:
Added some new stories, lots more to backfill using new sub-components. Would rather defer to a future PR.